De-duplicate FAQs (#1118)
XRPLF/xrpl-dev-portal: content/faq.md

#### Can currencies other than XRP be traded through XRPL?

Yes, the XRP Ledger was built specifically to be able to tokenize arbitrary assets, such as USD, EUR, Oil, Gold, reward points, and more. Any currency can be issued on the XRP Ledger. This is illustrated by XRPL’s growing community that backs a variety of fiat and crypto tokens.

#### Isn’t XRPL only for payments?

Although XRPL was initially developed for payment use cases, both the ledger and its native digital asset XRP are increasingly popular for a range of innovative blockchain use cases. New standards proposals for issuing NFTs, XRPL Labs’ hooks amendment for smart contract functionality, and federated sidechains are all currently works in progress.

## Validators and Unique Node Lists

#### What service do transaction validators provide?

> Validators determine if transactions meet protocol requirements, and are therefore “valid.” The service validators uniquely provide is administratively grouping transactions into ordered units, agreeing on one such ordering specifically to prevent double spending. <!-- STYLE_OVERRIDE: therefore -->

Reviewer: ```suggestion
All nodes ensure that transactions meet protocol requirements, and are therefore “valid.” The service that validators uniquely provide is administratively grouping transactions into ordered units, agreeing on one such ordering specifically to prevent double spending. <!-- STYLE_OVERRIDE: therefore -->
```

See [Consensus](consensus.html) and the [Ripple Labs Tech Talk: Understanding Consensus](https://ripple.com/insights/ripple-labs-tech-talk-consensus-within-the-ripple-protocol/) for more information about the consensus process.

#### How much does it cost to run a validator?

> Running a validator does not require any fees or XRP. It is comparable in cost to running an email server in terms of its electricity use.

Reviewer: ```suggestion
Running a validator does not require any fees or XRP. It is comparable in cost to running an email server in terms of electrical usage.
```

Author: I prefer the existing wording here. (Fun fact: searching "electrical usage" turns up a lot of results for "electricity use" instead.)

However, there can be only one latest _validated_ ledger version at any given time; other versions are irrelevant and harmless.

For more information about how the XRP Ledger's consensus mechanism behaves in adverse situations, see [Consensus Protections Against Attacks and Failure Modes](consensus-protections.html).

#### Does the XRP Ledger have a formal process for adding validators?

No, a formal process for adding validators is not compatible with the XRP Ledger, because it is a system with no central authority.

Publishers of individual default UNLs set their own policies for when to add or remove validators from their lists of recommendations.

For recommendations and best practices, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).

## Role of XRP

#### Why does Ripple use XRP holdings?

> Ripple's XRP holdings incentivize the company to make the XRP Ledger as useful as possible. XRP exists as a native asset in the XRP Ledger for anti-spam transaction purposes, and for currency bridging only if beneficial to users. Otherwise, XRP is completely optional.

Reviewer: ```suggestion
Ripple's XRP holdings incentivize the company to make the XRP Ledger as useful as possible. XRP exists as a native asset in the XRP Ledger for anti-spam transaction purposes, and for currency bridging only if beneficial to users. Otherwise, XRP is optional.
```

Ripple is also one of several entities who publish recommended UNL settings and runs validators on the network. As of mid-2021, Ripple runs 6 of the 35+ validators in the default UNL.

#### Does Ripple offer a secure method to download their software?

`rippled` source code is available at <https://github.com/ripple/rippled>, where the tip of the `master`, `release` and `develop` branches always contains a version-setting commit signed by a `rippled` developer. Ripple also publishes pre-built binary packages for CentOS, RedHat Enterprise Linux, Fedora, Ubuntu, and Debian Linux. Those packages are digitally signed so that they are tamper-evident and their authenticity can be verified. Lastly, release bulletins are made available over a secure website, and include the commit ID of the repository, as well as the cryptographic hash values of the packages that are published. <!-- STYLE_OVERRIDE: evident -->

#### Does the XRP Ledger distinguish between the codebase for validation and the one for user software?

> Yes. There are several [XRP Ledger client libraries](client-libraries.html) which are intended for users. These libraries have a different codebases and repositories from the [core XRP Ledger server](the-rippled-server.html) which powers the network and validates transactions.

Reviewer: ```suggestion
Yes. There are several [XRP Ledger client libraries](client-libraries.html) which are intended for user software developers. These libraries have different codebases and repositories from the [core XRP Ledger server](the-rippled-server.html) which powers the network and validates transactions.
```

Reviewer 2: I'm so confused it's not even funny maybe I should just delete everything
